File: tests/fixtures/code/sample.ts
// Greets someone by name.
function greet(name: string): string {
  const message: string = `Hello, ${name}!`;
  return message;
}

const friends: string[] = ["Ada", "Grace", "Margaret"];
for (const friend of friends) {
  console.log(greet(friend));
}

File: tests/fixtures/code/sample.go
// Greets someone by name.
package main

import "fmt"

func greet(name string) string {
	message := fmt.Sprintf("Hello, %s!", name)
	return message
}

func main() {
	friends := []string{"Ada", "Grace", "Margaret"}
	for _, friend := range friends {
		fmt.Println(greet(friend))
	}
}
